### DISCUSSION TITLE
Bitkub Thailand <> Akash Network Partnership Proposal

### DISCUSSION THREAD

**Discussion by hmlyptr** [akash-network/community#598] [General]
I am reaching out on behalf of Bitkub Thailand to discuss a collaboration opportunity that I believe will greatly benefit Akash Network in establishing a robust presence within the Thai community.

As Thailand’s number 1 SEC-regulated cryptocurrency exchange platform holding 95% of the market share, our company also diversifies in other products including Bitkub Chain and Bitkub NEXT. We have previously partnered with projects such as Circle, Tether, and Hedera, providing us the experience and expertise to support current and potential partners to gain increased visibility and awareness, trading volume boost, and on-site presence. 

We would love to discuss this potential collaboration further and explore how we can work together to achieve our shared objectives. Please let us know if you would be interested in exploring this opportunity further and we can arrange a meeting at your earliest convenience.

**Comment by rodri-r** [comment]
If you just want to list Akash in your exchange, you can list AKT token without asking anyone.

If you have other ideas for collabs, please share them here.
